src/main/java/dan200/computercraft/core/lua/CobaltLuaMachine.java

```diff
@@ -323,6 +324,63 @@ private LuaValue toValue( @Nullable Object object, @Nullable Map<Object, LuaValu
             return table;
         }
 
+        if( object instanceof CompoundNBT )
```

Comment:
`dan200.computercraft.core` generally should avoid having any hard dependencies on Minecraft - we want to be able to run it outside of Minecraft too (such as in an emulator). What we do elsewhere (such as within the `commands` API) is to [convert the NBT to plain a `Map<?, ?>`](https://github.com/SquidDev-CC/CC-Tweaked/blob/514db30fb1d40c5a73e70c191120513166f356ea/src/main/java/dan200/computercraft/shared/util/NBTUtil.java#L91) within the method itself.